lib/lotus/generators/application/container/config/.env.development.tt

```diff
@@ -1,2 +1,14 @@
 # Define ENV variables for development environment
+<%- if config[:adapter] == 'mysql' %>
+<%= config[:app_name].to_env_s %>_DATABASE_URL="mysql://localhost/<%= config[:app_name] %>_development"
```

Comment:
What do you think about DRY with `config[:adapter_uri]`? Furthermore, please ensure sqlite3 URI is escaped with Shellword